Optimization of SEO and a11y

kristofferlarberg opened the pull request:
This PR adds meta-data and adjusts the DOM structure and CSS to optimize for accessibility (especially for tabbing and screen reading.)

In order to get a full score for accessibility in Lighthouse tests the RenditionList needs to be structured in a different way. This is left for now because it's a little to big of an issue to handle within the scope of this assignment.

Correct meta-data also needs to be supplied by Tris.

https://user-images.githubusercontent.com/51208557/121673218-f6e21a80-cab0-11eb-8c64-6d3eb558a184.mov

calmfors commented on src/App.js:
```diff
@@ -31,8 +31,18 @@ const App = (props) => {
+        <meta name="twitter:description" content="Presentation of work by Tris Vonna-Michell." />
+        <meta name="twitter:image" content="https://images.prismic.io/trisvonnamichell/6392235a-5597-4bb1-aa05-21a37c33b122_TVM-audio+poems-09+copy+2.jpg" />
+        <link rel="shortcut icon" href="/favicon1.png" />
         <script
```
Should it be work or works by...?

kristofferlarberg replied:
If figure he'll give us the correct material, and we'll change it then?

calmfors commented on src/components/Nav.js:
```diff
@@ -39,15 +39,29 @@ const Title = styled(PageTitle)`
+            Info
+          </OpenInfoBoxButton>
+        </>
       )}
```
Better to make PageTitle a button, because it acts like a button (ie you click it and something happens).

kristofferlarberg replied:
I thought about that, but then the screen reader wont understand that it's the title of the page. And either way I think it's weird to open something by clicking on the name, so would like to pitch using a button, which should be something more interesting than this one ofc:) Maybe a symbol like the ones in the remote?

calmfors reviewed (commented):
Nice work! :)
Favicon is gone?
Background downscaling gone in GlobalStyle (probably my fault).

kristofferlarberg commented:
> Nice work! :)
> Favicon is gone?
> Background downscaling gone in GlobalStyle (probably my fault).

Thanks!

I changed the favicon so it's now a black square on `Home` and a black swuare with a white circle inside on `Renditions`. Like a shutter? Atm it doesn't change when going back via the stop button on the remote but otherwise works!

Ok! Sorry, I changed the structure somewhere around there so could be my fault, not looking closely in the merge conflicts..
